.github/actions/send-teams-notification/action.yml

```diff
@@ -4,70 +4,261 @@ inputs:
+      run: |
+        COLOR=808080
+        case $STATUS in
+          success)
+            COLOR=36A64F
+            ;;
+          failure)
+            COLOR=A30200
+            ;;
+        esac
+        echo "result=$COLOR" >> $GITHUB_OUTPUT
+
+    - name: Compute small sha
+      id: compute-small-sha
+      shell: bash
+      env:
+        GITHUB_LONG_SHA: ${{ github.sha }}
+      run: |
+        SMALL_SHA=${GITHUB_LONG_SHA:0:6}
+        echo "result=$SMALL_SHA" >> $GITHUB_OUTPUT
+
+    - name: Compute needs
+      id: compute-needs
+      shell: bash
+      env:
+        NEEDS: ${{ inputs.needs }}
+      run: |
+        NEEDS_OUTPUT=$(echo $NEEDS | jq -r 'to_entries[] | "\(.key): \(.value.result)"')
+        echo "result=$NEEDS_OUTPUT" >> $GITHUB_OUTPUT
+
+    - name: Compute message
+      id: compute-message
+      shell: bash
+      env:
+        EVENT_NAME: ${{ github.event_name }}
+        BLOCK_MESSAGE: ${{ inputs.message }}
+        APPEND: ${{ inputs.append }}
+        PR_TITLE: ${{ github.event.pull_request.title }}
+        ISSUE_BODY: ${{ github.event.issue.body }}
+        COMMIT_MESSAGE: ${{ github.event.head_commit.message }}
+        NEEDS: ${{ steps.compute-needs.outputs.result }}
+      run: |
+        ${{ github.action_path }}/compute-message.sh >> $GITHUB_OUTPUT
+
+    - name: Compute event
+      id: compute-event
+      shell: bash
+      env:
+        EVENT_NAME: ${{ github.event_name }}
+        REPO_URL: ${{ github.server_url }}/${{ github.repository }}
+        PR_NUMBER: ${{ github.event.pull_request.number }}
+        SHA: ${{ github.sha }}
+      run: |
+        RESULT="$EVENT_NAME"
+        if [[ "$EVENT_NAME" == 'pull_request' ]]; then
+          RESULT="[$EVENT_NAME #$PR_NUMBER]($REPO_URL/pull/$PR_NUMBER)"
+        elif [[ "$EVENT_NAME" == 'push' ]]; then
+          RESULT="[$EVENT_NAME]($REPO_URL/commit/$SHA/checks)"
+        fi
+        echo "result=$RESULT" >> $GITHUB_OUTPUT
+
+    - name: Compute run link
+      id: compute-run-link
+      shell: bash
+      env:
+        REPO_URL: ${{ github.server_url }}/${{ github.repository }}
+        WORKFLOW: ${{ github.workflow }}
+      run: |
+        RESULT="$REPO_URL/actions/runs/$GITHUB_RUN_ID/attempts/$GITHUB_RUN_ATTEMPT"
+        echo "result=$RESULT" >> $GITHUB_OUTPUT
+
+    - name: Compute workflow title
+      id: compute-workflow-title
+      shell: bash
+      run: |
+        RESULT="$GITHUB_WORKFLOW"
+        if [[ "$GITHUB_RUN_ATTEMPT" != '1' ]]; then
+          RESULT+=" (attempt #$GITHUB_RUN_ATTEMPT)"
+        fi
+        echo "result=$RESULT" >> $GITHUB_OUTPUT
+
+    - name: Get branch name
+      uses: Alfresco/alfresco-build-tools/.github/actions/get-branch-name@v8.4.0
+
     - name: Send teams notification
-      uses: skitionek/notify-microsoft-teams@9c67757f64d610fb6748d8ff3c11f284355ed7ec # v1.0.8
+      uses: skitionek/notify-microsoft-teams@190d4d92146df11f854709774a4dae6eaf5e2aa3 # master
```

Comment:
I'm a bit concerned about the state of this repository, it's not great with 2 commits and no releases in the last year lol

Reply:
Same, now I think we can always change and use another action if that's better